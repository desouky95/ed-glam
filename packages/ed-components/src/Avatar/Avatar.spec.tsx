import { create } from 'react-test-renderer';
import { Avatar } from '.';

it('render avatar', () => {
	const component = create(<Avatar />).toJSON();
	expect(component).toMatchSnapshot();
});
