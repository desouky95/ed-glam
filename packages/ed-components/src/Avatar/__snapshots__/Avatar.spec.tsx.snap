// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render avatar 1`] = `
<div
  className="sc-bczRLJ fHRskr"
  onClick={[Function]}
  shape="circle"
  size="small"
/>
`;
